### PR TITLE
[WIP][1.4 - DO NOT MERGE] Update version numbers for 1.4 release

### LINKED ISSUE #402: Update component versions for 1.4 release

### PR DESCRIPTION
**Pull request by btat**
Version numbering and values.yaml files will need to be adjusted depending on final release.

Resolves #402

### CODE REVIEW

**Comment by cjschroder** [comment]
:+1: I think you found everything.

File: xml/app_uaa_values_yaml.xml [hunk 1]
```diff
@@ -31,11 +32,15 @@ kube:
...
+  limits:
+    nproc:
+      hard: ""
+      soft: ""
+  organization: "splatform"
   psp:
     nonprivileged: ~
   registry:
-    hostname: "registry.suse.com"
+    hostname: "staging.registry.howdoi.website"
```
Comment (cjschroder) [inline xml/app_uaa_values_yaml.xml]: Is the the real literal hostname? we're using howdoi.website?

Comment (btat) [reply xml/app_uaa_values_yaml.xml]: I'll fix that and refrain from copy pasting till the file is finalized.

**Comment by cjschroder** [comment]
o look you found more. Nice catch with the Stratos chart version entity.

**Comment by cjschroder** [comment]
Oh sorry, I forgot you had opened this. I did a partial update, as the 1.4 release happened while you were gone https://github.com/SUSE/doc-cap/commit/6357064e3d4d11a912cf5744f86333e80fd7fa72

**Comment by btat** [comment]
Closing this one as it was superseded by #422

**Pull request closed without merging**